docs/support/prometheus.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Prometheus
+parent: Supporting Applications
+nav_order: 750
+---
+## Prometheus: System Monitoring
+
+There's a lot going on within the BitBox Base and it's good to have reliable metrics over time to aid in development, fine-tuning, during operations and when looking for the cause of a specific error.
+[Prometheus](https://prometheus.io/) takes care of that.
+It is a open-source monitoring solution that is easily extendable to monitor any application.
+Measurements are stored in a time-series database that can easily be queried and visualized, e.g. using [Grafana](grafana.md).
+This database is also the main data pool that is queried by the Base Middleware.
+
+### Installation
+
+The Prometheus archive file is downloaded directly from the GitHub releases page and checked against a hardcoded checksum.
+The archive is extracted, its content copied to various system locations and proper ownerwhip is set while building the Armbian image.
+
+```bash
+mkdir -p /etc/prometheus /var/lib/prometheus
+cp prometheus promtool /usr/local/bin/
+cp -r consoles/ console_libraries/ /etc/prometheus/
+chown -R prometheus /etc/prometheus /var/lib/prometheus
+```
+
+### Configuration
+
+Applications providing metrics basically just run a webserver that returns plain-text content in a specific format on a certain port.
+Prometheus queries these URI in fixed intervals and stores the metrics in its database.
+Where to get this information how often is specified in `/etc/prometheus/prometheus.yml`:
+
+```yaml
+global:
+  scrape_interval:     5m
+  evaluation_interval: 5m
+scrape_configs:
+  - job_name: node
+    scrape_interval: 1m
+    static_configs:
+      - targets: ['127.0.0.1:9100']
+  - job_name: base
+    scrape_interval: 1m
+    static_configs:
+      - targets: ['127.0.0.1:8400']
+  - job_name: bitcoind
+    static_configs:
+      - targets: ['127.0.0.1:8334']
+  - job_name: electrs
+    static_configs:
+    - targets: ['127.0.0.1:4224']
+  - job_name: lightningd
+    static_configs:
+    - targets: ['127.0.0.1:9900']
+```
+
+### Metrics
+
+The following metrics are collected both from the system and from specific applications.
+
+* **Operating System**: the [Prometheus Node Exporter](https://github.com/prometheus/node_exporter) collects very granualar system operations information.
```

Comment:
Nit: Maybe it'd be more readable to say "collects very granular information about the system, such as CPU and memory usage".

Reply:
ACK